server/conf/messages

```diff
@@ -144,6 +144,9 @@ link.programDetailsSr=Program details for {0}
 # Link text to an external site to read more about a program.
 link.externalLink=External site
 
+# Screen reader text next to a link indicating that it opens in a new tab.
```

Comment:
nit: How about something like: "Screen reader text for open-in-new icon indicating that link opens in a new tab." ?

Reply:
Done!